inscriptions/ISic000310.xml

```diff
@@ -185,60 +190,24 @@
-                    <expan>
-                        <abbr>q</abbr>
-                        <ex>uaestor</ex>
-                    </expan>
-                    <expan>
-                        <abbr>d</abbr>
-                        <ex>ecreto</ex>
-                    </expan>
-                    <expan>
-                        <abbr>d</abbr>
-                        <ex>ecurionum</ex>
-                    </expan>
-                    aedil
-                    <supplied reason="lost">is</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="3"/>pro hono
-                    <supplied reason="lost">re
-                        <choice>
-                            <sic>IIviratus</sic>
-                            <corr>duoviratus</corr>
-                        </choice>
-                    </supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">tr</supplied>iclin
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">- - -</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost"/>
-                    <lb n="5"/>t<supplied reason="lost">rib</supplied>us l
-                    <supplied reason="lost">ectis dedit</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="1"/><name><expan><abbr>Q</abbr><ex>uintus</ex></expan></name> <g ref="#interpunct">·</g> Lutati<supplied reason="lost">us</supplied><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><expan><abbr>q</abbr><ex>uaestor</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>ecreto</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>ecurionum</ex></expan> aedil<supplied reason="lost">is</supplied><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/>pro hono<supplied reason="lost">re</supplied><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">tr</supplied>iclin<gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">ium</supplied><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="5"/><gap reason="lost" extent="unknown" unit="character"/>t<supplied reason="lost">ri</supplied>bus l<supplied reason="lost">ectis</supplied><gap reason="lost" extent="unknown" unit="character"/>    
                 </ab>
-                <!--This EDR text 141128 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
+              
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>CIL</note></app></listApp>
```

Comment:
NB what you're presenting here is actually partly the txt of Manganaro 1989, not CIL, and you need to choose whether to report the text as it is reported in CIL from the antiquarian tradition, or to go with Manganaro's conjectural amendments of lines 4-5 (which is the text presented in EDR). I'm not sure there is any basis for thinking there is a gap at the start of either lines 4 or 5? I don't mind whether you go with CIL (in which case you'll need to put lines 4 and 5 in orig simply, as being unclear in significance) or Manganaro, but it needs to be a clear choice between them at this point. If you go with Manganaro, then what Mommsen reports after Carrera in lines 4-5 needs to be reported for each line in the apparatus.